src/Illuminate/Log/Logger.php

```diff
@@ -262,7 +262,7 @@ protected function formatMessage($message)
             return var_export($message->toArray(), true);
         }
 
-        return $message;
+        return (string) $message;
```

Comment:
I added this because the `fireLogEvent` always expects the `$message` to be a string.